compiler/noirc_frontend/src/tests/traits.rs

```diff
@@ -450,14 +448,13 @@ fn trait_alias_polymorphic_where_clause() {
 
         fn main() {
             assert(0.foo().bar().baz() == qux(0));
-                                          ^^^ No matching impl found for `T: Baz`
-                                          ~~~ No impl for `T: Baz`
         }
     "#;
 
     // TODO(https://github.com/noir-lang/noir/issues/6467)
     // assert_no_errors!(src);
-    check_errors!(src);
+    let errors = get_program_errors!(src);
+    assert!(!errors.is_empty());
```

Comment:
Why do we just assert that errors exist rather than their exact form now?

Reply:
It is worth mentioning Rust also doesn't support trait aliases in stable yet, which we do. Hence why we wanted to allow this code originally